src/main/java/org/jabref/model/entry/FileFieldParser.java

```diff
@@ -1,59 +1,58 @@
 package org.jabref.model.entry;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 
 public class FileFieldParser {
-    public static List<LinkedFile> parse(String value) {
-        if ((value == null) || value.trim().isEmpty()) {
-            return Collections.emptyList();
-        }
 
+    public static List<LinkedFile> parse(String value) {
```

Comment:
It's fine in this case, because the value of the file field is only stored as a String in `BibEntry`. On calling `getFiles` the String is converted into a list, but this list is not stored in `BibEntry`, so it's not possible to compromise its internal state.

You could call this a "defensive copy", which is fine.